src/winetricks

```diff
@@ -8416,8 +8490,9 @@ load_dotnet40()
     w_package_warn_win64
 
     case "$W_PLATFORM" in
-        windows_cmd) ;;
-        *) w_warn "dotnet40 does not yet fully work or install on wine.  Caveat emptor." ;;
+        windows_cygwin|windows_windows) ;;
+        nix_linux|nix_freebsd|nix_darwin|nix_redox) w_warn "dotnet40 does not yet fully work or install on wine.  Caveat emptor." ;;
+        *) w_die "Unknown platform '$W_PLATFORM' used in load_dotnet40()"
```

Comment:
Again, unnecessary

Reply:
Same reasoning as https://github.com/Winetricks/winetricks/pull/1338/files/7c0eed4e8a25108c4f9e345657356343a07cbf78#r332385126 for `*)`